rtsf-at-checkout-event-reconciler/events/process_events.go

```diff
@@ -34,98 +40,80 @@ var NextRFIDData = []RFIDEventEntry{}
-	if !ok {
-		edgexcontext.LoggingClient.Error("DeviceCV setting not found")
-		return false, nil
-	}
-	deviceRFID, ok := appsettings["DeviceRFID"]
-	if !ok {
-		edgexcontext.LoggingClient.Error("DeviceRFID setting not found")
-		return false, nil
-	}
+	deviceCV := eventsProcessing.ProcessConfig.DeviceCV
+
+	deviceRFID := eventsProcessing.ProcessConfig.DeviceRFID
 
-	result, _ := params[0].(models.Event)
+	result, _ := param.(models.Event)
 	for _, reading := range result.Readings {
-		eventName := reading.Name
-		edgexcontext.LoggingClient.Debug(fmt.Sprintf("Processing Checkout Event: %s", eventName))
+		readingData := reading.(models.ObjectReading)
+		eventName := readingData.ResourceName
+		lc.Debugf("Processing Checkout Event: %s", eventName)
 		eventOk := checkEventOrderValid(eventName, edgexcontext)
 		if !eventOk {
-			edgexcontext.LoggingClient.Error(fmt.Sprintf("Error: event occurred out of order: %v", eventName))
+			lc.Errorf("Error: event occurred out of order: %v", eventName)
 			continue
 		}
 
-		switch reading.Device {
-		case devicePos+"-rest", devicePos+"-mqtt":
-			processDevicePosReading(reading, edgexcontext)
+		switch readingData.DeviceName {
+		case devicePos + "-rest", devicePos + "-mqtt":
+			eventsProcessing.processDevicePosReading(readingData, edgexcontext)
 
-		case deviceScale, deviceScale+"-rest", deviceScale+"-mqtt":
-			processDeviceScaleReading(reading, edgexcontext)
+		case deviceScale, deviceScale + "-rest", deviceScale + "-mqtt":
+			eventsProcessing.processDeviceScaleReading(readingData, lc)
 
-		case deviceCV+"-rest", deviceCV+"-mqtt":
-			processDeviceCVReading(reading, edgexcontext)
+		case deviceCV + "-rest", deviceCV + "-mqtt":
+			eventsProcessing.processDeviceCVReading(readingData, lc)
 
-		case deviceRFID+"-rest", deviceRFID+"-mqtt":
-			processDeviceRFIDReading(reading, edgexcontext)
+		case deviceRFID + "-rest", deviceRFID + "-mqtt":
+			eventsProcessing.processDeviceRFIDReading(readingData, lc)
 
 		default:
-			edgexcontext.LoggingClient.Error(fmt.Sprintf("Did not recognize Device: %s", reading.Device))
+			lc.Errorf("Did not recognize Device: %s", readingData.DeviceName)
 			continue
 		}
 
 		msg := formatWebsocketMessage(eventName)
 		sendWebsocketMessage(msg, edgexcontext)
 	}
 
-	edgexcontext.LoggingClient.Trace(fmt.Sprintf("RTTLog: %v", RttlogData))
-	edgexcontext.LoggingClient.Trace(fmt.Sprintf("ScaleData: %v", ScaleData))
-	edgexcontext.LoggingClient.Trace(fmt.Sprintf("CvData: %v", CurrentCVData))
-	edgexcontext.LoggingClient.Trace(fmt.Sprintf("RfidData: %v", CurrentRFIDData))
+	lc.Tracef("RTTLog: %v", RttlogData)
+	lc.Tracef("ScaleData: %v", ScaleData)
+	lc.Tracef("CvData: %v", CurrentCVData)
+	lc.Tracef("RfidData: %v", CurrentRFIDData)
 
 	return false, nil
 }
 
-func processDeviceCVReading(reading models.Reading, edgexcontext *appcontext.Context) {
+func (eventsProcessing *EventsProcessor) processDeviceCVReading(reading models.ObjectReading, lc logger.LoggingClient) {
```

Comment:
Same for all these `process..` funcs
```suggestion
func (eventsProcessing *EventsProcessor) processDeviceCVReading(readingObjectValue interface{}, lc logger.LoggingClient) {
```